biodatasets/cellfinder/cellfinder.py

```diff
@@ -0,0 +1,226 @@
+(see https://www.informatik.hu-berlin.de/de/forschung/gebiete/wbi/resources/cellfinder/).
+"""
+from pathlib import Path
+from typing import Dict, Iterator, Tuple
+
+import datasets
+
+import utils.parsing as parsing
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@inproceedings{neves2012annotating,
+  title={Annotating and evaluating text for stem cell research},
+  author={Neves, Mariana and Damaschun, Alexander and Kurtz, Andreas and Leser, Ulf},
+  booktitle={Proceedings of the Third Workshop on Building and Evaluation Resources for Biomedical Text Mining\
+   (BioTxtM 2012) at Language Resources and Evaluation (LREC). Istanbul, Turkey},
+  pages={16--23},
+  year={2012},
+  organization={Citeseer}
+}
+"""
+
+_DATASETNAME = "cellfinder"
+
+_DESCRIPTION = """\
+The CellFinder project aims to create a stem cell data repository by linking information from existing public databases
+ and by performing text mining on the research literature. The first version of the corpus is composed
+ of 10 full text documents containing more than 2,100 sentences, 65,000 tokens and 5,200 annotations for entities.
+ The corpus has been annotated with six types of entities (anatomical parts, cell components, cell lines, cell types,
+ genes/protein and species) with an overall inter-annotator agreement around 80%.
+(see https://www.informatik.hu-berlin.de/de/forschung/gebiete/wbi/resources/cellfinder/).
+"""
+
+_HOMEPAGE = "https://www.informatik.hu-berlin.de/de/forschung/gebiete/wbi/resources/cellfinder/"
+_LICENSE = "CC BY-SA 3.0"
+
+_SOURCE_URL = "https://www.informatik.hu-berlin.de/de/forschung/gebiete/wbi/resources/cellfinder/"
+_URLS = {
+    _DATASETNAME: _SOURCE_URL + "cellfinder1_brat.tar.gz",
+    _DATASETNAME + "_splits": _SOURCE_URL + "cellfinder1_brat_sections.tar.gz",
+}
+
+_SUPPORTED_TASKS = [
+    Tasks.NAMED_ENTITY_RECOGNITION
+]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+_ENTITY_TYPES = [
+    "Anatomy",
+    "CellComponent",
+    "CellLine",
+    "CellType",
+    "GeneProtein",
+    "Species",
+]
+
+
+class CellFinderDataset(datasets.GeneratorBasedBuilder):
+    """The CellFinder corpus."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    BUILDER_CONFIGS = [
+        BigBioConfig(
+            name="cellfinder_source",
+            version=SOURCE_VERSION,
+            description="CellFinder source schema",
+            schema="source",
+            subset_id="cellfinder",
+        ),
+        BigBioConfig(
+            name="cellfinder_bigbio_kb",
+            version=BIGBIO_VERSION,
+            description="CellFinder BigBio schema",
+            schema="bigbio_kb",
+            subset_id="cellfinder",
+        ),
+        BigBioConfig(
+            name="cellfinder_source_splits",
```

Comment:
Thanks for adding the filtered version. Could you please change the subset_id like this? https://github.com/bigscience-workshop/biomedical/blob/master/biodatasets/medmentions/medmentions.py#L116

That is `celfinder_source_splits` -> `cellfinder_splits_source`

This is the standard formatting, otherwise we cannot run tests!